website/packages/docs/src/pages/pkg-babel-plugin-strip-runtime.mdx

```diff
@@ -29,29 +29,26 @@ Configure in your Babel config.
 }
 ```
 
-### onFoundStyleRules: (rules: string[]) => void
-
-Will callback at the end of a file pass with all found style rules.
-
-### compiledRequireExclude: boolean
+#### compiledRequireExclude
 
 When set will prevent additional require (one import per rule) in the bundle and add style rules to meta data.
 This could be used when enabling style sheet extraction in a different configuration for SSR.
 
-Defaults to `false`.
-
-### extractStylesToDirectory
+- Type: `boolean`
```

Comment:
Do we want this same "type: boolean, default: false" elsewhere?  I think it's missing on like `extractStylesToDirectory`, but a bit hard to parse.  No real ask, just a comment.

Reply:
ye sure thing!